Update the smart contract documentation to v1 (#434)
Concordium/concordium.github.io: source/shared/smart-contracts/guides/setup-contract.rst

      [dependencies]
      concordium-std = { path = "./path/to/concordium-std" }

.. seealso::

>    It is possible to build smart contracts without the use for Rust's ``std``.

Author: ```suggestion
   It is possible to build smart contracts without using Rust's ``std``.
```
